doc-botonic/docs/components/components-whatsapptemp.md

```diff
@@ -0,0 +1,45 @@
+---
+id: whatsapptemp
+title: WhatsApp Template
+---
+
+The `whatsapptemplate` component allows you to send WhatsApp templates in the bot actions.
+
+## Properties
+
+| Property  | Type   | Description                                                                                                                                                                                                                                      | Required | Default value |
+|-----------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| name      | string | Name of the template                                                                                                                                                                                                                             | Yes      | -             |
+| namespace | string | Namespace of the template. Beginning with v2.27.8, this must be the namespace associated with the WhatsApp business account that owns the phone number associated with the current WhatsApp Business API client or the message will fail to send | Yes      | -             |
+| language  | string | Language of the template                                                                                                                                                                                                                         | Yes      | -             |
+| header    | object | Header of the template                                                                                                                                                                                                                           | No       | -             |
+| body      | object | Body of the template                                                                                                                                                                                                                             | No       | -             |
+| footer    | object | Footer of the template                                                                                                                                                                                                                           | No       | -             |
+
+## Example
+
+Example of `./actions/example-whatsappTemplate.js` 
```

Comment:
--> `./actions/example-whatsapp-template.js`: this is the standard we follow for naming files